### PR TITLE
Translating `en/model_doc` docs to Japanese.

### LINKED ISSUE #27392: [i18n-JP] Translating `en/model_doc` docs to Japanese
Hi!

Let's bring the documentation to all the Japanese-speaking community 🌐 

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

* Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
* Please translate in a gender-neutral way.
* Add your translations to the folder called `ja` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
* Register your translation in `ja/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @stevhliu and @MKhalusova for review.
* 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Model_doc section
- [x] [albert.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/albert.md) #27401
- [x]  [align.md.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/align.md) #27401
- [x]  [altclip.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/altclip.md) #27401
- [x]  [audio-spectrogram-transformer.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/audio-spectrogram-transformer.md) #27401
- [x]  [auto.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/auto.md) #27401
- [x]  [autoformer.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/autoformer.md) #27401
- [x]  [bark.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bark.md) #27264
- [x]  [bart.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bart.md) #27264
- [x]  [barthez.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/barthez.md) #27264
- [x]  [bartpho.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bartpho.md) #27264
- [x]  [beit.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/beit.md) #27264
- [x]  [bert-generation.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bert-generation.md) #27264
- [x]  [bert-japanese.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bert-japanese.md) #27264
- [x]  [bert.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bert.md) [](#27264)
- [x]  [bertweet.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bertweet.md) #27264
- [x]  [big_bird.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/big_bird.md) #27401
- [x]  [bigbird_pegasus.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bigbird_pegasus.md) #27264
- [ ]  [biogpt.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/biogpt.md) #27264
- [ ]  [bit.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bit.md) #27264
- [ ]  [blenderbot-small.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/blenderbot-small.md) #27264
- [ ]  [blenderbot.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/blenderbot.md) #27264

Keep on adding more as you go 🔥

### PR DESCRIPTION
**Pull request by Yuki-Imajuku**
# What does this PR do?
translating `en/model_doc` docs to Japanese.

Fixes #27392 

## Before submitting

- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@stevhliu and @MKhalusova

### CODE REVIEW

File: docs/source/ja/model_doc/audio-spectrogram-transformer.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/model_doc/audio_spectogram_transformer_architecture.png"
+alt="drawing" width="600"/>
+
+<small> Audio pectrogram Transformerのアーキテクチャ。<a href="https://arxiv.org/abs/2104.01778">元論文</a>より抜粋。</small>
```
Comment (stevhliu) [inline docs/source/ja/model_doc/audio-spectrogram-transformer.md:30]: Would you mind also fixing this in the `en` version? 🙂 

```suggestion
<small> Audio Spectrogram Transformerのアーキテクチャ。<a href="https://arxiv.org/abs/2104.01778">元論文</a>より抜粋。</small>
```

File: docs/source/ja/model_doc/autoformer.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+このモデルは[elisim](https://huggingface.co/elisim)と[kashif](https://huggingface.co/kashif)より提供されました。
+オリジナルのコードは[こちら](https://github.com/thuml/Autoformer)で見ることができます。
+
+## 参考資料
+
+Autoformerの使用を開始するのに役立つ公式のHugging Faceおよびコミュニティ（🌎で示されている）の参考資料の一覧です。ここに参考資料を提出したい場合は、気兼ねなくPull Requestを開いてください。私たちはそれをレビューいたします！参考資料は、既存のものを複製するのではなく、何か新しいことを示すことが理想的です。
+
+HuggingFaceのブログでAutoformerに関するブログ記事をチェックしてください：はい、トランスフォーマーは時系列予測に効果的です（+ Autoformer）
```
Comment (stevhliu) [inline docs/source/ja/model_doc/autoformer.md:36]: Duplicate sentence?

```suggestion
```

File: docs/source/ja/_toctree.yml [hunk 3]
```diff
@@ -245,6 +245,505 @@
     - local: main_classes/image_processor
       title: 画像処理プロセッサ
     title: 主要なクラス
+  - sections:
+    - isExpanded: false
+      sections:
+      - local: model_doc/albert
+        title: ALBERT
+      # - local: model_doc/bart
```
Comment (stevhliu) [inline docs/source/ja/_toctree.yml:253]: Not exactly sure why the doc-builder preview link is not working, but can you maybe try removing all the untranslated models from the `toctree` and see if that helps?

**Review by stevhliu — COMMENTED** [review]
Thanks for starting to translate the model doc APIs!

File: docs/source/ja/model_doc/audio-spectrogram-transformer.md (hunk 1 above)
Comment (Yuki-Imajuku) [reply docs/source/ja/model_doc/audio-spectrogram-transformer.md]: Sure!

File: docs/source/ja/_toctree.yml (hunk 3 above)
Comment (Yuki-Imajuku) [reply docs/source/ja/_toctree.yml:253]: OK! But it's possible that an indentation error in L102 may have affected it, so let me just check that first.

Comment (Yuki-Imajuku) [reply docs/source/ja/_toctree.yml]: Also, I got a 500 Error in [https://huggingface.co/docs/transformers/v4.35.1/ja/index](https://huggingface.co/docs/transformers/v4.35.1/ja/index)

**Review by stevhliu — APPROVED** [review]
LGTM, thanks for your contribution! ❤️

File: docs/source/ja/_toctree.yml (hunk 3 above)
Comment (stevhliu) [reply docs/source/ja/_toctree.yml]: I'm able to preview the docs [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_27401/ja/model_doc/albert), but I'm not sure why there's a 500 on the main docs. I'll check with the team :)

**Pull request merged**